apps/root/src/common/components/transaction-steps/index.tsx

```diff
@@ -436,14 +436,24 @@ const buildApproveTokenItem = ({
               {hasLoadReceipt && (
                 <TransactionStepSuccessLabel
                   label={
-                    <FormattedMessage
-                      description="approveAmount"
-                      defaultMessage="Approve {amount} {symbol}"
-                      values={{
-                        symbol: token.symbol,
-                        amount: formatCurrencyAmount({ amount, token, sigFigs: 4, intl }),
-                      }}
-                    />
+                    isPermit2Enabled ? (
```

Comment:
This should only be based on the approved amount, don't we have that data?

Reply:
We always have the max amount to approve, based on the quotes received. We dont have the `maxUint256` here. If permit2Enabled, we don't send an amount and the walletService submits unlimited

I know we are handling this different from txEvents, but I don't see how the user could have permit2Enabled, and don't submit an unlimited approval

Reply:
This is as of now, if we later want to enable users to exact approve permit2 contracts this will break. 

Just as a note, we used to allow users to exact approve the permit2 contract. Nothings says we will enable it again at some point, lets plan for the future :)